src/queries.ts

```diff
@@ -707,6 +707,9 @@ export const subjectPageQuery = gql`
       }
       allTopics: topics(all: true) {
         ...TopicInfo
+        article {
+          ...ArticleInfo
+        }
```

Comment:
Her henter du alle artikler for alle topics i et subject ved lasting av et subject. Alt av data hentes fra søkeindeks så det er ikkje så ille, men likevel ganske masse data. Og kun for å ha artikkelobjektet for verktøykassa. Når du klikker på en emneknapp i sida så har du jo artikkelen tilgjengelig i den topicen som finnes der. Hadde det gått an å bruke det objektet direkte i stedet for å måtte hente dobbelt så masse data?